### PR TITLE
Move creating description from label to CocinaObjectStore

### LINKED ISSUE #3625: Move setting description when absent to CocinaObjectStore
https://github.com/sul-dlss/dor-services-app/blob/main/app/services/cocina/object_creator.rb#L157

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3625

## Why was this change made? 🤔
This action is required regardless of datastore.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file system, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

Unit

### CODE REVIEW

File: app/services/cocina_object_store.rb [hunk 1]
```diff
@@ -349,7 +350,7 @@ def sync_from_symphony(cocina_request_object, druid)
     return cocina_request_object if catkeys.blank?
 
     result = RefreshMetadataAction.run(identifiers: catkeys, cocina_object: cocina_request_object, druid: druid)
-    return if result.failure?
+    return cocina_request_object if result.failure?
```
Comment (jmartin-sul) [inline app/services/cocina_object_store.rb:353]: i don't think i understand the change here.  fix for an unrelated issue, or related to description in a way i'm not picking up on?

**Review by ndushay — APPROVED** [review]
again, not fully grokking, but approving.

**Review by jmartin-sul — APPROVED** [review]
this lgtm, a bit hesitant to merge since i'm not sure i fully grok the change, but seems fine afaict

File: app/services/cocina_object_store.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina_object_store.rb:353]: I'll be honest; I don't recall.

**Pull request merged**